### ISSUE TITLE
Action randomly failing on First but there is no first in pr

### ISSUE THREAD

**Issue by albertoxamin** [mheap/require-checklist-action#44]
this is the template
```md
## Context

## Test Plan
> Detail the steps to test your changes thoroughly. Include both manual and automated testing as applicable.
> Specify **only** the environment, data, and tools **needed** for testing.
> Can include screenshots, gifs, or videos to demonstrate the changes.
> Useful for reviewers to understand the changes and for QA to test the changes.


## Checklist for All Changes
- [ ] This PR is linked to a linear ticket if applicable (tick also if not applicable)
- [ ] I will manually test the change during [QA](https://www.notion.so/xxxx/xxxx?v=xxxxx)
- [ ] I have followed the [best practices](https://www.notion.so/xxxx/Pull-Request-Best-practices-xxx)


## Revert Plan
<!-- Describe the steps to revert your changes in case it's necessary.
Include any special considerations or potential issues related to the reversion process. -->

```

this is the error in the logs
```
2023-12-18T09:25:59.8335516Z Download action repository 'mheap/require-checklist-action@v2' (SHA:1baf7cfc5be24da7bb1939c4034eecf928a56492)
2023-12-18T09:26:00.6057974Z Complete job name: Best Practices - PR Description and Checklist
2023-12-18T09:26:00.7300825Z ##[group]Run mheap/require-checklist-action@v2
2023-12-18T09:26:00.7301750Z with:
2023-12-18T09:26:00.7302195Z   requireChecklist: false
2023-12-18T09:26:00.7303172Z   token: ***
2023-12-18T09:26:00.7303614Z ##[endgroup]
2023-12-18T09:26:01.7457319Z Completed task list item: This PR is linked to a linear ticket if applicable (tick also if not applicable)
2023-12-18T09:26:01.7460717Z Completed task list item: I will manually test the change during [QA](https://www.notion.so/xxxxx/xxxxx?v=xxxxx)
2023-12-18T09:26:01.7465116Z Completed task list item: I have followed the [best practices](https://www.notion.so/xxxx/Pull-Request-Best-practices)
2023-12-18T09:26:01.7468613Z Incomplete task list item: First
2023-12-18T09:26:01.7507013Z ##[error]The following items are not marked as completed: First
2023-12-18T09:26:01.7941260Z Cleaning up orphan processes
```

**Comment by mheap** [comment]
@albertoxamin Can you link me to the GH Actions run for this failure? The phrase "First" isn't anywhere in this repo, so I'm not sure where it's coming from

**Comment by albertoxamin** [comment]
I know, I am also very confused, this is the job

```
name: Require Checklist

on:
  pull_request:
    types: [opened, edited, synchronize]
  issues:
    types: [opened, edited, deleted]

jobs:
  check-appropriate-pr-description-and-checklist:
    name: Best Practices - PR Description and Checklist
    runs-on: ubuntu-latest
    steps:
      - uses: mheap/require-checklist-action@v2
        with:
          requireChecklist: false # If this is true and there are no checklists detected, the action will fail

```

the run output

https://pipelinesghubeus11.actions.githubusercontent.com/h046VmRMfqHzdmHsmyAZNpvLQgVQB2KwtKIi3l7ZCjtVsFn4d0/_apis/pipelines/1/runs/21650/signedlogcontent/2?urlExpires=2023-12-18T11%3A17%3A43.0507728Z&urlSigningMethod=HMACV1&urlSignature=2DZA%2FO4ZUcgy7hozOSyy9o3IDs5P3aX7S4WepJyBrUE%3D

**Comment by mheap** [comment]
Is there any commented out content in the PR body? How about the comments?

**Comment by albertoxamin** [comment]
This is the exact content

```md
## Context

Allow searching by either decision or entity id.

## Test Plan
> Detail the steps to test your changes thoroughly. Include both manual and automated testing as applicable.
> Specify **only** the environment, data, and tools **needed** for testing.
> Can include screenshots, gifs, or videos to demonstrate the changes.
> Useful for reviewers to understand the changes and for QA to test the changes.


## Checklist for All Changes
- [x] This PR is linked to a linear ticket if applicable (tick also if not applicable)
- [x] I will manually test the change during [QA](https://www.notion.so/taktile/01bfa254ceb741e9b670c5930084bebd?v=708197f3e4af49bb833f5ff0fe8c49ee)
- [x] I have followed the [best practices](https://www.notion.so/taktile/Pull-Request-Best-practices-52062d2ac4a046faa10c89ec2756b62b)


## Revert Plan
<!-- Describe the steps to revert your changes in case it's necessary.
Include any special considerations or potential issues related to the reversion process. -->

```

**Comment by tomlarkworthy** [comment]
the issue is all comments are checked, not just the PR description. In our case, our linear intergration is adding a comment contained a checklist called "First". You can close this issue, its out fault.

**Comment by mheap** [comment]
Thanks for debugging @tomlarkworthy. Would an option to disable checking comments be useful?

**Comment by tomlarkworthy** [comment]
Yeah for us it is somewhat unexpected behaviour and something we would turn off if available

**Comment by mheap** [comment]
There's a new `skipComments` input available in 2.3.0. You can target `mheap/require-checklist-action@v2` and set the following to skip comments:

```yaml
with:
  skipComments: "true"
```
